PupDates-CrossPol/pupdates_frontend
# Sk splash

#### What's this PR do?

Adds basic styling to the splash page, including logo, inputs for email and password, and a non-functional login button.

#### Where should the reviewer start?

There's only an App component for now. Also, be sure to run ```npm i``` because I had to install even more packages for the fonts and linear gradients.

#### How should this be manually tested?

Start the local server and make sure it looks okay either on your phone or on a simulator. I've looked at it on my iPhone6S, an Android (I believe a Pixel 3?), and an iPhone XR

#### Any background context you want to provide?

Apparently you cannot change the styling of text for a Button, so our "button" is a TouchableOpacity component. Also, the font load is an asynchronous function that doesn't always complete quickly, so it throws errors. If you dismiss the errors, then you can see the app without the imported font. We should look into a workaround for this.

Also, couldn't quite figure out how to give the inputs shadow rather than borders, so I went with light grey borders for now and can be tweaked as we continue to work and learn more about styling props.

#### What are the relevant tickets?

Issue#7, but not closed.

#### Screenshots (if appropriate)

(the android one has a weird symbol thing on it, ignore that, it came from the screenshot controls?)

<img width="395" alt="Screen Shot 2019-12-17 at 8 55 03 PM" src="https://user-images.githubusercontent.com/46407593/71054771-65d6f100-2110-11ea-84ff-092273152134.png">
<img width="382" alt="Screen Shot 2019-12-17 at 8 51 43 PM" src="https://user-images.githubusercontent.com/46407593/71054775-6a030e80-2110-11ea-96f0-0a22756511d7.png">

#### Questions:

Colors aren't perfect, but I was just picking things in the dev tools. Does XD happen to give hex codes? Also, our logo seems to look slightly cut off no matter how I size it...am I doing something wrong?
